src/cargo_shim/mod.rs

```diff
@@ -774,7 +791,7 @@ impl BuildConfig {
                 match output {
                     CargoOutput::Message( message ) => {
                         match self.message_format {
-                            MessageFormat::Human => diagnostic_formatter::print( self.use_color, &message ),
+                            MessageFormat::Human | MessageFormat::Other => diagnostic_formatter::print( self.use_color, &message ),
```

Comment:
Add a `MessageFormat::__Nonexhaustive => unreachable!()` match arm here (and in other `match`es) instead.